extensions/wrapper/wrapper-ee-api/src/main/java/de/sovity/edc/ext/wrapper/api/ee/EnterpriseEditionResource.java

```diff
@@ -15,23 +15,44 @@
 package de.sovity.edc.ext.wrapper.api.ee;
 
 import de.sovity.edc.ext.wrapper.api.ee.model.ConnectorLimits;
+import de.sovity.edc.ext.wrapper.api.ee.model.StoredFile;
 import io.swagger.v3.oas.annotations.Operation;
 import io.swagger.v3.oas.annotations.tags.Tag;
-import jakarta.ws.rs.GET;
-import jakarta.ws.rs.Path;
-import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.*;
 import jakarta.ws.rs.core.MediaType;
 
+import java.util.ArrayList;
+import java.util.Map;
+
 /**
  * Our EDC Enterprise Edition API Endpoints to be included in our generated EDC API Wrapper Clients
  */
 @Path("wrapper/ee")
 @Tag(name = "Enterprise Edition", description = "EDC Enterprise Edition API Endpoints. Included here for ease of use.")
 public interface EnterpriseEditionResource {
-
     @GET
     @Path("connector-limits")
     @Produces(MediaType.APPLICATION_JSON)
     @Operation(description = "Available and used resources of a connector.")
     ConnectorLimits connectorLimits();
+
+    @POST
+    @Path("file-storage/files}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Operation(description = "Upload a file to the file storage.")
+    StoredFile blobStoreUploadFile(String fileName, byte[] data);
```

Comment:
this request should be a multipart request as described before

See https://howtodoinjava.com/jersey/jax-rs-jersey-ajax-multi-file-upload-example/

This is not a file upload endpoint